examples/tensorflow-mnist-pixel-threshold/task-plugins/securingai_custom/evaluation/pixelthreshold.py

```diff
@@ -14,31 +14,28 @@
 #
 # ACCESS THE FULL CC BY 4.0 LICENSE HERE:
 # https://creativecommons.org/licenses/by/4.0/legalcode
-"""A task plugin module for the Pixel Threshold evasion attack.
 
+"""A task plugin module for the Pixel Threshold evasion attack.
 The Pixel Threshold [kotyan2019]_ is an evasion attack that attempts to
 fool a trained classifier by perturbing a test image within a certain threshold
 of pixels. This task plugin uses the Adversarial Robustness Toolbox's
 [art2019]_ implementation of the Pixel Threshold attack |pt_art|.
```

Comment:
```suggestion
"""A task plugin module for the Pixel Threshold evasion attack.

The Pixel Threshold [kotyan2019]_ is an evasion attack that attempts to
fool a trained classifier by perturbing a test image within a certain threshold
of pixels. This task plugin uses the Adversarial Robustness Toolbox's
[art2019]_ implementation of the Pixel Threshold attack |pt_art|.

```
We want to keep the whitespace here to separate the sections so that the restructured text parser doesn't get confused.